### PR TITLE
Initial Changes for MP JWT 2.1 For Issue #21186

### PR DESCRIPTION
**Pull request by simysc**
For issue #21186

### CODE REVIEW

**Comment by simysc** [comment]
#build

File: dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#CMVCPATHNAME com.ibm.ws.security/resources/io/openliberty/security/mp/jwt/v21/resources/MpConfigProxyMessages.nlsprops
+#COMPONENTPREFIX CWWKS
+#COMPONENTNAMEFOR WebSphere Application Server MicroProfile JWT 2.1 Config Proxy
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+# Message prefix block: CWWKS5780 - CWWKS5785
+
+MPJWT_12_CONFIG_PROXY_PROCESSED=CWWKS5780I: The MicroProfile JWT version 2.1 mpConfigProxy processed successfully.
```
Comment (ramkumar-k-9286) [inline dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops]:
```suggestion
MPJWT_12_CONFIG_PROXY_PROCESSED=CWWKS5780I: The MicroProfile JWT version 2.1 mpConfigProxy was processed successfully.
```

File: dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+MPJWT_12_CONFIG_PROXY_PROCESSED.explanation=The indicated configuration  processed successfully.
```
Comment (ramkumar-k-9286) [inline dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops]:
```suggestion
MPJWT_12_CONFIG_PROXY_PROCESSED.explanation=The indicated configuration was processed successfully.
```

File: dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+MPJWT_12_CONFIG_PROXY_PROCESSED.useraction=No action is required.
+
+MPJWT_12_CONFIG_PROXY_MODIFIED=CWWKS5781I: The MicroProfile JWT version 2.1 mpConfigProxy processed successfully.
```
Comment (ramkumar-k-9286) [inline dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops]:
```suggestion
MPJWT_12_CONFIG_PROXY_MODIFIED=CWWKS5781I: The MicroProfile JWT version 2.1 mpConfigProxy was processed successfully.
```

File: dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+MPJWT_12_CONFIG_PROXY_MODIFIED.explanation=The indicated configuration processed successfully.
```
Comment (ramkumar-k-9286) [inline dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops]:
```suggestion
MPJWT_12_CONFIG_PROXY_MODIFIED.explanation=The indicated configuration was processed successfully.
```

File: dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+MPJWT_12_CONFIG_PROXY_MODIFIED.useraction=No action is required.
+
+MPJWT_12_CONFIG_PROXY_DEACTIVATED=CWWKS5782I: The MicroProfile JWT version 2.1 mpConfigProxy deactivated successfully.
```
Comment (ramkumar-k-9286) [inline dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops]:
```suggestion
MPJWT_12_CONFIG_PROXY_DEACTIVATED=CWWKS5782I: The MicroProfile JWT version 2.1 mpConfigProxy was deactivated successfully.
```

File: dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+MPJWT_12_CONFIG_PROXY_DEACTIVATED.explanation=The indicated configuration deactivated successfully.
```
Comment (ramkumar-k-9286) [inline dev/io.openliberty.security.mp.jwt.2.1.config/resources/io/openliberty/security/mp/jwt/v21/config/resources/MpConfigProxyMessages.nlsprops]:
```suggestion
MPJWT_12_CONFIG_PROXY_DEACTIVATED.explanation=The indicated configuration was deactivated successfully.
```

**Comment by simysc** [comment]
#build

**Comment by simysc** [comment]
#libby

**Review by teddyjtorres — APPROVED** [review]
Thank you Simy for these changes.

**Pull request merged**